.github/workflows/build_test_deploy.yml

```diff
@@ -1245,6 +1245,47 @@ jobs:
         run: cd packages/next-swc && cargo nextest run -p next-dev-tests --release --no-fail-fast
         if: ${{needs.build.outputs.docsChange == 'nope'}}
 
+  test-bench-native-integration:
+    name: Bench Integration Test Native Code
```

Comment:
@sokra  This fails when we publish a new version

```
npm ERR! notarget No matching version found for next@13.3.1.
```

https://github.com/vercel/next.js/actions/runs/4769164009/jobs/8479317012#step:7:21

Should we skip in that case?

Reply:
Yeah, it's currently using the latest canary in benchmarks. So it's failing during publish since the version in package.json is already incremented but it's not yet published.
We need to improve that to use the local packaged build of next.js instead, but i was too lazy for that...